packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -29,8 +29,10 @@ export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>,
   value?: string;
   /** Array of attribute values */
   attributes?: string[] | SearchAttribute[];
+  /* Additional elements added after the attributes in the form */
```

Comment:
Can you revert the SearchInput changes? They're not quite the same as #6043

Reply:
ups sorry, did not realize by branch was dirty,